forms-flow-web/src/components/FOI/FOIRequest/MinistryReview/MinistryReview.js

```diff
@@ -227,8 +227,12 @@ const MinistryReview = React.memo(({ userDetail }) => {
   const [originalDivisions, setOriginalDivisions] = React.useState([]);
   const [hasReceivedDate, setHasReceivedDate] = React.useState(true);
   const [isMinistryRestricted, setIsMinistryRestricted] = useState(false);
-  const [isMCFPersonal, setIsMCFPersonal] = useState(bcgovcode.replaceAll('"', '') == "MCF" && requestDetails.requestType == FOI_COMPONENT_CONSTANTS.REQUEST_TYPE_PERSONAL);
-
+  const [isMCFPersonal, setIsMCFPersonal] = useState(
+    bcgovcode.replaceAll('"', "") == "MCF" &&
+      requestDetails.requestType ==
+        FOI_COMPONENT_CONSTANTS.REQUEST_TYPE_PERSONAL
+  );
+  const [unSavedRequest, setUnSavedRequest] = React.useState(false);
```

Comment:
moved unSavedRequest state variable declaration to the top to fix ministryview load issue